<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ErrorPanel</class>
  <widget class="QMainWindow" name="ErrorPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>362</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>LDView Errors &amp; Warnings</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <widget class="QWidget">
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <layout class="QVBoxLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item>
              <widget class="QTreeWidget" name="errorListView">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>3</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                </property>
                <column>
                  <property name="text">
                    <string>Errors</string>
                  </property>
                  <property name="clickable">
                    <bool>false</bool>
                  </property>
                  <property name="resizable">
                    <bool>false</bool>
                  </property>
                </column>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="showWarningsButton">
                <property name="text">
                  <string>Show Warnings</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QVBoxLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item>
              <layout class="QHBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="TextLabel1">
                    <property name="text">
                      <string>Show the following errors:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer1_2">
                    <property name="sizeHint">
                      <size>
                        <width>16</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item>
              <layout class="QHBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>0</number>
                </property>
                <item>
                  <spacer name="Spacer1">
                    <property name="sizeHint">
                      <size>
                        <width>16</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>6</number>
                    </property>
                    <item>
                      <widget class="QCheckBox" name="generalErrorButton">
                        <property name="text">
                          <string>General error</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="parseErrorButton">
                        <property name="text">
                          <string>Parse error</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="fileNotFoundButton">
                        <property name="text">
                          <string>File not found</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="singularMatrixButton">
                        <property name="text">
                          <string>Singular matrix</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="partDeterminantButton">
                        <property name="text">
                          <string>Part transformed non-uniform</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="nonFlatQuadButton">
                        <property name="text">
                          <string>Non-flat quad</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="concaveQuadButton">
                        <property name="text">
                          <string>Concave quad</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="badVertexOrderButton">
                        <property name="text">
                          <string>Bad vertex sequence</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="identicalVerticesButton">
                        <property name="text">
                          <string>Identical vertices</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="colinearPointsButton">
                        <property name="text">
                          <string>Co-linear points</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="BFCWarningButton">
                        <property name="text">
                          <string>BFC warning</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="BFCErrorButton">
                        <property name="text">
                          <string>BFC error</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="MPDErrorButton">
                        <property name="text">
                          <string>MPD error</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="whitespaceButton">
                        <property name="text">
                          <string>Whitespace</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="partrenamedButton">
                        <property name="text">
                          <string>Part renamed</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="unofficialPartButton">
                        <property name="text">
                          <string>Unofficial part used</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="modelLoopButton">
                        <property name="text">
                          <string>Model loop</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="metaCommandProblemButton">
                        <property name="text">
                          <string>Meta Command problem</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="tooManyRequestsButton">
                        <property name="text">
                          <string>Too many requests</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item>
              <spacer name="Spacer3">
                <property name="sizeHint">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
            <item>
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QPushButton" name="showAllButton">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Show All</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer35">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QPushButton" name="showNoneButton">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Show None</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer36">
                    <property name="sizeHint">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
